diff --git a/kubernetes/oneclick/createAll.bash b/kubernetes/oneclick/createAll.bash
index 488f920..a197ae0 100755
--- a/kubernetes/oneclick/createAll.bash
+++ b/kubernetes/oneclick/createAll.bash
@@ -85,7 +85,7 @@ create_onap_helm() {
 
   # assign default auth token
   if [[ -z $ONAP_DEFAULT_AUTH_TOKEN ]]; then
-    DEFAULT_SECRET=`kubectl get secrets -n $1 | grep default-token |  awk '{ print $1}'`
+    DEFAULT_SECRET=`kubectl get secrets -n $1 | grep default-token |  awk '{ print $1}' | head -n 1`
     ONAP_DEFAULT_AUTH_TOKEN=`kubectl get secrets $DEFAULT_SECRET -n $1 -o yaml | grep  'token:'  | awk '{ print $2}' | base64 --decode`
   fi
 
